src/System.Runtime.Extensions/tests/System/AppDomainTests.cs

```diff
@@ -268,28 +316,43 @@ public void IsCompatibilitySwitchSet()
         [Fact]
         public void IsDefaultAppDomain()
         {
-            Assert.True(AppDomain.CurrentDomain.IsDefaultAppDomain());
+            RemoteInvoke(() =>
```

Comment:
Is a RemoteExecutor required here because the xunit appdomain where the tests are running in isn't the default one?